src/senaite/jsonapi/api.py

```diff
@@ -199,7 +199,7 @@ def update_items(portal_type=None, uid=None, endpoint=None, **kw):
         if not is_update_allowed(obj):
             fail(401, "Update of {} is not allowed".format(api.get_path(obj)))
 
-        obj = update_object_with_data(obj, record)
+        obj = update_object_with_data(obj, **record)
```

Comment:
Hi @toropok,
I don't understand this change.

The signature of `update_object_with_data` looks like this:

```python
def update_object_with_data(content, record):
    """Update the content with the record data

    :param content: A single folderish catalog brain or content object
    :type content: ATContentType/DexterityContentType/CatalogBrain
    :param record: The data to update
    :type record: dict
    :returns: The updated content object
    :rtype: object
    :raises:
        APIError,
        :class:`~plone.jsonapi.routes.exceptions.APIError`
    """
```

Where `record` should be a `dict` and not keyword arguments.